src/order.rs

```diff
@@ -145,6 +146,19 @@ impl BtcDaiOrder {
     }
 }
 
+impl Display for BtcDaiOrder {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        write!(
+            f,
+            "Position: {}, BTC: {}, DAI: {}",
+            self.position,
+            self.base.amount.as_btc(),
```

Comment:
This is not needed, the amount already implement a proper `std::fmt::Display` that includes the currency symbol.